CHANGES.txt

```diff
@@ -4,8 +4,15 @@ Changes
 1.2 (unreleased)
 ----------------
 
-- Nothing changed yet.
+* NOTE: this is the last version to officially support Python 2.
 
+* Start in test coverage.
+
+* Fix a bug in `ImportDatabase()`.
+
+* Cleanups.
```

Comment:
(I would not bother mentioning externally-invisible details like this in a changelog.)

Reply:
fixed